modules/mono/glue/Managed/Files/GD.cs

```diff
@@ -110,6 +110,31 @@ public static void PrintT(params object[] what)
             godot_icall_GD_printt(what);
         }
 
+        public static double Randf()
+        {
+            return godot_icall_GD_randf();
+        }
+
+        public static uint Randi()
+        {
+            return godot_icall_GD_randi();
+        }
+
+        public static void Randomize()
+        {
+            godot_icall_GD_randomize();
+        }
+
+        public static double rand_range(double from, double to)
+        {
+            return godot_icall_GD_rand_range(from, to);
+        }
+
+        public static uint RandSeed(ulong seed, out ulong newSeed)
+        {
+            return godot_icall_GD_rand_seed(seed, out newSeed);
```

Comment:
Shouldn't it be `godot_icall_GD_rand_from_seed` and `RandFromSeed`, as in core? The current name suggests that it returns a random seed, while it returns a random number from the given seed, and also gives the updated seed.

Reply:
I agree `RandFromSeed` would be better but the current name matches the GDScript one.